src/main/webapp/app/list-of-complaints/list-of-complaints.route.ts

```diff
@@ -10,6 +11,17 @@ export const listOfComplaintsRoute: Routes = [
         data: {
             authorities: ['ROLE_ADMIN', 'ROLE_INSTRUCTOR', 'ROLE_TA'],
             pageTitle: 'artemisApp.complaint.listOfComplaints.title',
+            complaintType: ComplaintType.COMPLAINT,
+        },
+        canActivate: [UserRouteAccessService],
+    },
+    {
+        path: 'moreFeedbackRequests',
```

Comment:
I think you should use more-feedback-requests here to be consistent with other routes